gpiozero/devices.py

```diff
@@ -272,7 +279,10 @@ def _default_pin_factory():
                 try:
                     mod_name, cls_name = entry_point.split(':', 1)
                     module = __import__(mod_name, fromlist=(cls_name,))
-                    return getattr(module, cls_name)()
+                    pin_factory = getattr(module, cls_name)()
+                    if name == 'native':
+                        warnings.warn(native_fallback_message, NativePinFactoryFallback)
```

Comment:
For consistency with all the other warnings in gpiozero, should we stick with the `warnings.warn(NativePinFactoryFallback(native_fallback_message))` pattern, at least for now?

Reply:
Done